docs/UserGuide.md

```diff
@@ -385,6 +393,49 @@ Examples:
 
 <br>
 
+### Editing a patient’s multi-valued attributes
+
+<div markdown="block" class="alert alert-success">
+
+:bulb: **Tip:** You can understand the commands about muti-valued attributes as follows.
+
+#### Adding a value to an attribute: `add` `-p`
+
+You can add a value to a patient's attribute with the `add` command.
+
+Format: **`add`**`-p PATIENT_INDEX xyz/XYZ_VALUE`
+
+Example:
+* `list` followed by `add -p 1 t/high-risk` adds the `high-risk` tag (attribute `t`) to the 1st patient in the patient list.
+
+#### Editing a value of an attribute: `edit` `-p` `-xyz`
+
+Format: **`edit`**`-p PATIENT_INDEX -xyz XYZ_INDEX xyz/XYZ_VALUE`
+
+Example:
+* `list` followed by `edit -p 2 -t 3 t/high-risk` edits the 3rd tag (attribute `t`) of the 2nd patient in the patient list to `high-risk`.
+
+
+#### Deleting a value from an attribute: `edit` `-p` `-xyz`
+
+Format: **`delete`**`-p PATIENT_INDEX -xyz XYZ_INDEX`
+
+Example:
+* `list` followed by `delete -p 2 -t 3` deletes the 3rd tag (attribute `t`) of the 2nd patient in the patient list.
+
+<div markdown="block" class="alert alert-info">
```

Comment:
I'm not sure how to properly insert the `xyz` format explanation to the UG since it doesn't seem to fit anywhere (so I ended up putting it in the box). The box in box itself behaves normally (i.e., box in box with different colors).